web_src/js/utils/dom.js

```diff
@@ -258,16 +258,27 @@ export function isElemVisible(element) {
   return Boolean(element.offsetWidth || element.offsetHeight || element.getClientRects().length);
 }
 
+export function getComboMarkdownEditor(el) {
+  if (el?.jquery) el = el[0];
```

Comment:
What is `.jquery`?

Reply:
`.jquery` displays the jquery version, or null if el is not a jquery element

Reply:
Use `el instanceof $` instead.

Reply:
If jquery is not imported, this writing method will cause an error.

Reply:
Could just import it above, but seeing that we are in the middle of removing jQuery, I would suggest this function to only accept `HTMLElement`.

Reply:
solved